content/python/encyclopedia/replace-in-python.md

```diff
@@ -11,7 +11,7 @@ Catalog Content:
 ---
 -->
 
```

Comment:
Add a title:

# Replace in Python

Reply:
We're putting the titles in the metadata now! No need to add this.

Reply:
sounds good, thanks!